cuckoo/pid_guid_map.py

```diff
@@ -34,7 +34,7 @@ def __init__(self, pid: int, start_time: float, end_time: float, guid: str = Non
         if not guid:
             self.guid = self.assign_guid()
         else:
-            self.guid: str = str(UUID(guid))
+            self.guid: str = f"{{{str(UUID(guid))}}}"
```

Comment:
? Seems weird

Reply:
That is how you get {} in f strings without being evaluated, gotta use curly braces back to back